### DISCUSSION TITLE
Lagged stream using DJI Drone (either mini 3 Pro or 300 RTK) or even ffmpeg

### DISCUSSION THREAD

**Discussion by antonymarion** [OvenMediaLabs/OvenMediaEngine#1691] [Help]
**Describe the bug**

**To Reproduce**
1. default server.xml configuration
2. See error in logs
![image](https://github.com/user-attachments/assets/2ce21afa-a467-4147-8bd3-2b03db0c5024)

**Expected behavior**
The stream is very laggy and I have no idea why

**Logs**
cf above

**Server (please complete the following information):**
 - OvenMediaEngine Last Version over Docker inside Portainer

**Player (please complete the following information):**
 - Device: any
 - OS: any
 - Browser Chrome
 - Version: last

**Additional context**
tested with stream sent over ffmpeg, seeing very lagged video too, was ok before. After few seconds over ffmpeg it seems better. Logs are like these on server side with ffmpeg stream (not red any more)
![image](https://github.com/user-attachments/assets/73de60fb-48b2-4d77-8570-40975f122336)

cmd used to send over ffmpeg: `ffmpeg -d -stream_loop -1 -re -i ./my_video.mp4 -c copy -f flv -y rtmp://ome.stationdrone.net:1935/app/test`

**N.B.:** testing with SRS does not show the PB. SRS is running inside docker in a Portainer too.

**Comment by antonymarion** [comment]
Just to be sure, I tested sending my test stream using ffmpeg on the demo space, and I reproduced the same laggy effect as in our portainer over OVH:

![image](https://github.com/user-attachments/assets/df484ae8-2744-426f-9731-e9155ead3b92)

**Comment by antonymarion** [comment]
Examples of video files sent using ffmpeg and showing lags: 


https://github.com/user-attachments/assets/43a77448-8055-4a15-b4f1-be0483f09201

For the above video, I remember that previous tests before showed better results compared to SRS (since SRS does not handle b-frames), but now it does not show better results

https://we.tl/t-tRi3pYwx2P

**Comment by getroot** [comment]
What protocol are you playing with? WebRTC or llhls?

**Comment by antonymarion** [comment]
Webrtc.

I then disabled llhls on server side in the xml

**Comment by getroot** [comment]
Does your sample video have b-frames? Browsers do not support b-frames in WebRTC streams. It is a limitation of the browser, not a limitation of OME.
OME provides a transcoder, so if you encode the stream, it can be played normally.
On the other hand, LLHLS supports b-frames.

**Reply by antonymarion** [reply]
Exactly it works perfectly on FF thks

**Reply by antonymarion** [reply]
(so issue on Chrome side then)

**Reply by antonymarion** [reply]
Thk you so much for you swift reply
